plugins/base/src/test/kotlin/renderers/html/TextStylesTest.kt

```diff
@@ -69,6 +69,14 @@ class TextStylesTest : HtmlRenderingOnlyTestBase() {
         files.contents.getValue("test-page.html").contains("&nbsp;<strike><b>styled text</b></strike>")
     }
 
+    @Test
+    fun `should include blockquote`() {
+        val page = testPage {
+            text("blockquote text", styles = setOf(TextStyle.Quotation))
+        }
```

Comment:
I think this test will fail since we only wrap groups and not individual text nodes. I believe it should be like this:
```kotlin
testPage {
    group(styles = setOf(TextStyle.Quotation)) {
        text("blockquote text")
    }
}
```